docs/topics/elementary_programming/exercises.md

```diff
@@ -37,6 +37,17 @@ console.log('Programming is Fun!')
 
 </details>
 
+<details>
+  <summary>C# Solution:</summary>
+
+```cs linenums="1"
```

Comment:
Did the "cs" successfully add code highlighting?

Reply:
It did not. `csharp`, not `cs`, is the correct syntax. This has been fixed.